### PR TITLE
[1LP][RFR] Add 5.9 to the test_db_migrate_replication

### PR DESCRIPTION
**Pull request by jaryn**
Purpose or Intent
=================
Start testing with 5.9 db backup.

juwatts:

{{ pytest: -v cfme/tests/test_db_migrate.py::test_db_migrate_replication[59]  --use-provider complete }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8e4fad4f5fa06f744a7353f478f389b92220ab30:

`cfme/tests/test_db_migrate.py`:
- :bangbang: Line 133:101: **E501** *line too long (101 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4b1796db0d55acdce35be6ea01435d350314b7ce:

`cfme/tests/test_db_migrate.py`:
- :bangbang: Line 135:9: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by digitronik** [comment]
@jaryn please check  failures. For now  moving to WIPTEST

**Comment by dajoRH** [comment]
Lint report for commit ff3e5c8c895cf13acc78190d14467332aefbafa3:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 221:57: **E226** *missing whitespace around arithmetic operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/appliance/__init__.py [hunk 1]
```diff
@@ -217,7 +218,7 @@ def configure_appliance_internal(self, region, dbhostname, username, password, d
         self._run("--region {region} --internal --hostname {dbhostname} --username {username}"
             " --password {password} --dbname {dbname} --verbose --dbdisk {dbdisk}".format(
                 region=region, dbhostname=dbhostname, username=username, password=password,
-                dbname=dbname, dbdisk=dbdisk))
+                dbname=dbname, dbdisk=dbdisk), timeout=4 * 60)
```
Comment (jawatts) [inline cfme/utils/appliance/__init__.py]: Why pass 4*60 and not 240 here?

Comment (izapolsk) [reply cfme/utils/appliance/__init__.py]: 4*60 is more readable. but I'd rather advice using string value '4m'  if there is wait_for behind.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bee511ab9c6f86283e996e55cc5fd4f9bf3f814d:

`cfme/utils/ssh.py`:
- :red_circle: Line 225:13: **F821** *undefined name 'wait_for'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/appliance/__init__.py (hunk 1 above)
Comment (jaryn) [reply cfme/utils/appliance/__init__.py]: @izapolsk it is not possible to pass "4m". Float is required.

**Pull request merged**